requirements.txt

```diff
@@ -1,2 +1,3 @@
-llnl-thicket
-ipython
\ No newline at end of file
+llnl-thicket[]
```

Comment:
do you need blank brackets?

Reply:
no, it's not needed, that was an accident!